src/nexuslims_logger/db_logger_gui.py

```diff
@@ -303,7 +304,7 @@ def __init__(self, db_logger, screen_res=None):
         self.end_icon = PhotoImage(file=resource_path('window-close.png'))
         self.end_button = Button(self.button_frame,
                                  # takefocus="",
-                                 text="End session",
+                                 text="End session",               
```

Comment:
the trailing spaces are unnecessary, please remove them.

Reply:
![image](https://user-images.githubusercontent.com/61523149/123001770-b3bc5d00-d376-11eb-8be7-c89347359d94.png)